.travis.yml

```diff
@@ -2,7 +2,17 @@ language: generic
 
 script: bash ./deploy.sh
 
+branches:
+  only:
+    - gh-pages
```

Comment:
I think we want master in the only list also, or general PRs might not cause Travis to run and check that the files compile.